compiler/quilt/nodes.py

```diff
@@ -67,6 +67,10 @@ def __repr__(self):
         data_info = '\n'.join(sorted(self._data_keys()))
         return '%s\n%s%s' % (pinfo, group_info, data_info)
 
+    def __iter__(self):
+        for child in self._group_keys():
```

Comment:
We should actually return all nodes, not just groups. (I've commented on the bug report, too.)